service_api.py

```diff
@@ -54,39 +52,39 @@ def retreiveFile(url, destFolder, obj):
 
     path = os.path.join(destFolder, file_name)
     try:
-        resp = get_legacy_session().get(url=url, verify=False, stream=True, timeout=10)
+        resp = get_legacy_session().get(url=url, verify=False, stream=True)
         if str(resp.status_code) == '404':
+            resp.close()
             return False, "Plik nie istnieje"
```

Comment:
Czy jeżeli dobrze rozumiem to w tym fragmencie nie powinien iść kod dalej przez to że jest odpowiedź False?